src/Components/JobsContainer/Internal/JobsDialog/JobsDialog.stories.tsx

```diff
@@ -7,7 +7,7 @@ import { IJobsDialogProps } from './JobsDialog.types';
 const wrapperStyle = { width: '100%', height: '600px', padding: 8 };
 
 export default {
-    title: 'Components/JobsDialog',
+    title: 'Components/Jobs/JobsDialog',
```

Comment:
Since this is going to be a whole "app" of it's own, let's have it's own collection of components. For OAT we did the same thing. "Components - Jobs" would probably make sense here.